evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -165,7 +172,13 @@ def predict(self, X, y=None):
         """
         X, y = self._manage_woodwork(X, y)
         fh_ = self._set_forecast(X)
-        X = X.select_dtypes(exclude=["datetime64"])
+        X = X.ww.select(exclude=["Datetime"])
```

Comment:
should we call `self._remove_datetime` here as well? Unsure if we need the other functionaility in `self._remove_datetime`

Reply:
Great question! I think we could but that would cause an extra copy so I'd rather keep as is?

Reply:
sounds good to me!